### PR TITLE
148: (WIP) Adding blog listing to homepage, improving blog-listing component

### PR DESCRIPTION
**Pull request by astolfivincent**
Steps to test
- Go to Homepage

### CODE REVIEW

**Comment by rcaracaus** [comment]
Shouldn't there only be 3 blogs?

File: app/index.twig [hunk 1]
```diff
@@ -1,10 +1,7 @@
 {% extends "../app/layout.twig" %}
 
 {% block content %}
-
     {%  include "../components/hero-background-video/hero-background-video.twig" %}
-
-    {%  include "../components/home-service/home-service.twig" %}
-
+    {% include "../components/blog-listing/blog-listing.twig" with {'limit': '3'} %}
```
Comment (edisch) [inline app/index.twig]: Do we need this if we are feeding the a spliced array of 3 blogs in anyway?

Comment (edisch) [inline app/index.twig]: Do we need this if we are feeding a spliced array of 3 blogs in to this template anyway?

File: app.js [hunk 2]
```diff
@@ -1,76 +1,96 @@
...
-        if (index <= 2) {
-            bloglistingarr.push(mar['data']);
-        }
+        simplePage.simplePage(blogPage);
+        console.log(bloglistingarr);
+        var homePage = {
+            templatePath: 'app/index.twig',
+            renderedTemplatePath: 'public/index.html',
+            templateData: {
+                'blogData' : bloglistingarr.slice(0, 3), // trim listing on homepage down to 3
```
Comment (edisch) [inline app.js]: We spliced the blog listings to 3 here so we might not need the limit handlebars helper.

**Pull request merged**